metsim/disaggregate.py

```diff
@@ -13,7 +29,19 @@
 
 def disaggregate(df_daily, params, solar_geom):
     """
-    TODO
+    Take a daily timeseries and scale it down to a finer
+    time scale.
+
+    Args:
+        df_daily: Dataframe containing daily timeseries.
+            Should be the result of one of the methods
+            provided in the `methods` directory.
+        params: A dictionary containing the class parameters
+            of the MetSim object.
+        solar_geom: A dictionary of solar geometry variables
+
+    Returns:
+        A dataframe with sub-daily timeseries.
```

Comment:
So this would look like:

```    
    Parameters
    ----------
    df_daily : pd.DataFrame
        DataFrame containing daily timeseries with variables: (fill in variables).
        Should be the result of one of the methods provided in the `methods`
        directory.
    params : dict
        A dictionary containing the class parameters of the MetSim object.
    solar_geom : dict
        A dictionary of solar geometry variables

    Returns
    -------
    df_disagg : pd.DataFrame
        A dataframe with sub-daily timeseries.```